prompb/types.proto

```diff
@@ -49,3 +49,8 @@ message LabelMatcher {
   string name  = 2;
   string value = 3;
 }
+
+message ReadHints {
+  int64 step_ms = 1; // Query step size in milliseconds.
+  string func = 2;   // String representation of surrounding function or aggregation.
```

Comment:
Could it be any function? Or we should limit it to some small set common for remote storages? Maybe enum will be better?

Reply:
It's only a hint, and it can be any function.

Reply:
@AlekSi string vs enum was discussed at #2580, this PR follows that